eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/VariantEntityRepositoryFilter.java

```diff
@@ -68,42 +68,6 @@ public Criteria createCriteria() {
         return criteria;
     }
 
-    public static List<VariantRepositoryFilter> getRepositoryFilters(String maf, String polyphenScore, String siftScore,
-                                                                     List<String> studies, List<String> consequenceType) {
-        List<VariantRepositoryFilter> filters = new ArrayList<>();
-
-        VariantRepositoryFilter filter;
-
-        if (maf != null) {
-            filter = new VariantRepositoryFilter<>("st.maf", getValueFromRelation(maf), getRelationalOperatorFromRelation(maf));
-            filters.add(filter);
-        }
-        if (polyphenScore != null) {
-            filter = new VariantRepositoryFilter<>("annot.ct.polyphen.sc", getValueFromRelation(polyphenScore),
-                                                   getRelationalOperatorFromRelation(polyphenScore));
-            filters.add(filter);
-        }
-        if (siftScore != null) {
-            filter = new VariantRepositoryFilter<>("annot.ct.sift.sc", getValueFromRelation(siftScore),
-                                                   getRelationalOperatorFromRelation(siftScore));
-            filters.add(filter);
-        }
-        if (studies != null && !studies.isEmpty()) {
-            filter = new VariantRepositoryFilter<>("files.sid", studies, VariantEntityRepository.RelationalOperator.IN);
-            filters.add(filter);
-        }
-        if (consequenceType != null && !consequenceType.isEmpty()) {
-            List<Integer> consequenceTypeConv = consequenceType.stream()
-                                                               .map(c -> Integer
-                                                                       .parseInt(c.replaceAll("[^\\d.]", ""), 10))
-                                                               .collect(Collectors.toList());
-            filter = new VariantRepositoryFilter<>("annot.ct.so", consequenceTypeConv,
-                                                   VariantEntityRepository.RelationalOperator.IN);
-            filters.add(filter);
-        }
-        return filters;
-    }
-
     static Double getValueFromRelation(String relation) {
```

Comment:
protected static maybe instead of package public?

Maybe this could also be a normal protected non static method if we add in the class hierarchy a double filter that is a parent class to all the filters of type double.